lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1543,3 +1594,29 @@ int parse_periodic_search_pattern(const char *const pattern_str,
 
 	return 0;
 }
+
+int lte_lc_rai_set(void)
```

Comment:
This function is named like functions in the public API. Other functions in the helpers module do not have the `lte_lc` prefix, so I think it could be removed from this one as well to avoid confusion. I think `rai_set()` or `rai_configure()` would be enough.

Reply:
Renamed.